sympy/discrete/transforms.py

```diff
@@ -141,7 +141,7 @@ def _number_theoretic_transform(seq, prime, inverse=False):
                         "Number Theoretic Transform")
 
     a = [as_int(x) % p for x in seq]
-
+    
```

Comment:
Why have you added extra whitespace here? This causes flake8 code quality test to fail